src/core/execute.h

```diff
@@ -216,6 +219,12 @@ struct ExecContext {
         bool no_new_privileges_set:1;
 };
 
+static inline bool exec_context_restrict_namespaces_set(const ExecContext *c) {
+        assert(c);
+
+        return (c->restrict_namespaces & NAMESPACE_FLAGS_ALL) != NAMESPACE_FLAGS_ALL;
```

Comment:
`c->restrict_namespaces != NAMESPACE_FLAGS_ALL` should be equivalent, but maybe this version is better against future changes...